### PR TITLE
bump geany to 1.34.1, geany-plugins to 1.34

### PR DESCRIPTION
**Pull request by hoewweken**
Update geany to current version.
Minimal testing done, looks ok up to know.
Please review.
Regards,
Stephan

### CODE REVIEW

File: components/editor/geany-plugins/geany-plugins.p5m [hunk 1]
```diff
@@ -22,17 +23,12 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-<transform file path=usr/share/icons/.* -> default restart_fmri svc:/application/desktop-cache/icon-cache:default>
-<transform file path=usr/share/locale/([^/]+)(\..+){0,1}(/.+){0,1} -> default facet.locale.%<\1> true>
```
Comment (Mno-hime) [inline components/editor/geany-plugins/geany-plugins.p5m]: What's the reationale for this?

**Comment by Mno-hime** [comment]
Better separate it to one commit per component.

**Comment by hoewweken** [comment]
It was proposed to update the plugins also, so i put it in.
No special reason for this p5m thing, there have been many Differences between the sample-manifest and the geany-plugins.p5m that i just *blindly copied it over...
Maybe i should have merged that differences..

So i should do this:
close this PR
make 2 separate ones
recreate the geany-plugins.p5m in a better manner with the diff from the sample-manifest
...

**Comment by Mno-hime** [comment]
> It was proposed to update the plugins also, so i put it in.
> No special reason for this p5m thing, there have been many Differences between the sample-manifest and the geany-plugins.p5m that i just *blindly copied it over...
> Maybe i should have merged that differences..

Yes, one should merge differences between old and new sample-manifest, where appropriate.

> So i should do this:
> close this PR
> make 2 separate ones
> recreate the geany-plugins.p5m in a better manner with the diff from the sample-manifest

In essence. For that use branches, it will make your life easier as then you can switch between different PRs and work on them independently.

**Pull request closed without merging**